.eslintrc.js

```diff
@@ -7,4 +7,8 @@
 
 module.exports = {
   extends: './node_modules/eslint-config-hackreactor/index.js'
-};
\ No newline at end of file
+};
+
+"env": {
+  "es6": true
+}
```

Comment:
What is the purpose of this code ?Were there issues when these configs were not there??

Reply:
Hi Barsha, yes the eslinter was giving me `Parsing error: Unexpected token function` error when using correct syntax for ES6 async await. Please see the solution I found [here](https://stackoverflow.com/questions/50285821/eslint-parsing-error-unexpected-token-function-with-async).

Reply:
Thanks for clarification.